### PR TITLE
Added feature for opening issues via Slack. Markdown not supported

### LINKED ISSUE #40: Opening Issues from Slack
## Description
As a user
I need to create an issue from slack
so that I can automate issue creation.

### Update [Required]
- [ ] Add functionality to create an issue from Slack.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 hours

### PR DESCRIPTION
**Pull request by sammy1997**
# Description  
Feature to open issues from Slack. Markdown type body not supported yet. Issues can be opened with slash command /sysbot_open_issue <repo-name> <Author username> *Title of issue( in stars)* <body>

Fixes #40 

# Type of Change:
- Code
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
test issue

# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

### CODE REVIEW

File: slack_functions.py [hunk 1]
```diff
@@ -164,3 +164,25 @@ def claim_issue_slack(data):
...
+        return
+    issue_title = title_body_tokens[1]
+    issue_body = title_body_tokens[2]
+    status = open_issue_github(org_repo_owner, tokens[0], issue_title, issue_body, tokens[1])
+    if status == 201:
+        #If issue has been opened successfully
+        send_message_to_channels(channel_id, MESSAGE.get('success_issue', ''))
+    else:
+        send_message_to_channels(channel_id, MESSAGE.get('error_issue', ''))
+    return
```
Comment (sara-02) [inline slack_functions.py]: No need to add a `return` at the end.

**Comment by ramitsawhney27** [comment]
Sammy, you can make the change and I’ll merge it then.

**Comment by sammy1997** [comment]
Ok. I will update this

**Comment by sammy1997** [comment]
Updated

**Comment by sara-02** [comment]
:+1:

**Pull request merged**